src/apps/Permissions/Home/Apps.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { CardLayout } from '@aragon/ui'
+import { AppType } from '../../../prop-types'
+import AppCard from '../AppCard'
```

Comment:
`AppCard` is only used here right? Maybe move it to this folder?

Reply:
Done https://github.com/aragon/aragon/pull/899/commits/efd370ce36103ed07a488f5ba54890da648e140c